### DISCUSSION TITLE
System.Runtime, Version=6.0.0.0

### DISCUSSION THREAD

**Discussion by cristian304** [community/community#130345] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Good morning i have a problem with  one plugin for LSPDFR

problem is this

Could Not Load File Or Assembly 'System.Runtime, Version=6.0.0.0

I hope to find a solution as soon as possible.

**Comment by MorrisrCodes** [comment]
Did you try uninstalling and reinstalling dependencies?

**Reply by cristian304** [reply]
Good morning i have resolve with delete file newtonsoft.json of Main folder. Thanks

**Comment by cristian304** [comment]
I have resolve. Thanks
